de.dashup.application/src/test/resources/de.dashup.test/change_panel_structure.feature

```diff
@@ -0,0 +1,78 @@
+Feature: Change Panel Structure
+
+  As a basic user
+  I want to customize the central dashboard of dashup
+  In order to design dashup the way I want
+
+  Background: Authenticated
+    Given User is located on login page
+    When User submits e-mail and password
+    Then User is logged in
+
+    Scenario: Enable edit mode
+      Given User is located on the central dashboard
+      When User clicks on layout icon
+      Then Central dashboard can now be edited
+
+    Scenario: Add section
+      Given User enabled edit mode
+      When User clicks on add section button
+      Then New section is added with a placeholder name
+
+    Scenario: Modify section
+      Given User enabled edit mode
+      When User clicks on the name of a section
+      And User enters "my new panels"
+      And User unfocuses section name
+      Then Section will be renamed to "my new panels"
+
+    Scenario: Remove section
+      Given User enabled edit mode
+      When User clicks on the remove icon of a section
+      Then Section will be removed with all its containing panels
+
+#    Scenario: Reorder panels
```

Comment:
You could add these tests as well or am I wrong? I know, we do have unit tests for this, but anyways frontend and backend tests are separated. and independent

Reply:
In the moment it is not possible to execute tests because of database changes, there will be a new issue for testing